package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "transact-sdk-javascript",
+  "version": "0.0.0-alpha",
```

Comment:
This should probably have the initial version.  `0.1.0`?

Reply:
updated